src/Orleans.Core.Abstractions/Statistics/IHostEnvironmentStatistics.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Orleans.Statistics
+{
+    public interface IHostEnvironmentStatistics
+    {
+        long TotalPhysicalMemory { get; }
+
+        float CpuUsage { get; }
```

Comment:
Prefer if these names were suffixed with their units, like Bytes/Percentage/Proportion. Is CpuUsage 0-100 or 0-1?

Reply:
I agree with you. I wanted to not change too much previously existing naming, since they are used elsewhere. But if everybody is ok, I can update it everywhere

Reply:
I'm ok either way